contracts/marketplace/src/multitest.rs

```diff
@@ -3554,6 +3554,7 @@ fn try_remove_stale_ask() {
 
     // Transfer NFT from creator to owner. Creates a stale ask that needs to be updated
```

Comment:
might want to test the other cases of stale ask like expired / no approval, token burn.

Reply:
burning case is already covered by our tests
https://github.com/public-awesome/marketplace/blob/main/contracts/marketplace/src/multitest.rs#L3517-L3527